src/swap-helpers.ts

```diff
@@ -38,12 +39,14 @@ export interface SwapOrder {
   expirationDate?: Date
   preTx?: EdgeTransaction
   metadataNotes?: string
+  customNetworkFee?: JsonObject
```

Comment:
We should not need to pass this in again, since the `spendInfo` already has it. It should be up to the `fetchSwapPluginInner` methods to preserve the fees and stick them into the `spendInfo` as needed.